drivers/hmis/app/models/hmis/filter/enrollment_filter.rb

```diff
@@ -92,13 +96,13 @@ def with_household_tasks(scope)
       SQL
 
       # Clause for checking whether an Assessment falls within the "due period". There are due cases because the due period may be this year or last year.
-      this_year_annual_in_range = start_date.lteq(Date.current).and(cas_t[:assessment_date].gteq(start_date).and(cas_t[:assessment_date].lteq(end_date)))
-      last_year_annual_in_range = start_date.gt(Date.current).and(cas_t[:assessment_date].gteq(last_start_date).and(cas_t[:assessment_date].lteq(last_end_date)))
+      this_year_annual_in_range = start_date.lteq(today).and(cas_t[:assessment_date].gteq(start_date).and(cas_t[:assessment_date].lteq(end_date)))
+      last_year_annual_in_range = start_date.gt(today).and(cas_t[:assessment_date].gteq(last_start_date).and(cas_t[:assessment_date].lteq(last_end_date)))
       annual_in_range = this_year_annual_in_range.or(last_year_annual_in_range)
 
       # Clause for checking whether an Enrollment's Entry Date falls before the "anniverary". There are due cases because the anniversary may be this year or last year.
```

Comment:
```suggestion
      # Clause for checking whether an Enrollment's Entry Date falls before the "anniverary". There are two cases because the anniversary may be this year or last year.
```